debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -682,6 +693,27 @@ public static EventDeserializationFailureHandlingMode parse(String value) {
                                                                    .withDefault(DEFAULT_BINLOG_BUFFER_SIZE)
                                                                    .withValidation(Field::isNonNegativeInteger);
 
+    public static final Field BINLOG_READER_HEARTBEAT_INTERVAL = Field.create("binlog.reader.heartbeat.interval")
```

Comment:
The option name makes me think whether we can make this all a bit more generic and apply it to all the connectors. It's a general theme we should pursue, let's get away from the connector-specific implementations where it's reasonably possible.

Also in the case we probably can encapsulate most of it behind a facade in _debezium-core_ (e.g. timeout handling, production of the heartbeat message and passing it over to the event queue). The individual handlers then just would call this facade: `heartbeatController.onChangeEvent()` or something like that, and they'd instantiate the heartbeat controller with a method reference to the enqueue method.